runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -150,6 +155,7 @@
 	private Instant startScroll = Instant.now();
 	private String rememberedSearch;
 	private boolean waitSearchTick;
+	private boolean initializedOnce;
```

Comment:
remove this

Reply:
Disagree, but I will revert this only on the basis that it doesn't directly relate to this thread, but it makes improvements which I documented in 4333553, and this should be added in the future if not now.

Reply:
The listed code is no longer relevant and is unused.